lib/json_api_client/middleware/status.rb

```diff
@@ -28,8 +28,12 @@ def handle_status(code, env)
           raise Errors::NotFound, env[:url]
         when 409
           raise Errors::Conflict, env
+        when 429
+          raise Errors::ApiError, env
         when 400..499
           # some other error
+          # TODO: raise Errors::ApiError, env
```

Comment:
ok, I think we can actually add this back in. I'm pretty sure if you run the tests on master, 7 tests will fail, and its the same 7 tests that fail when you uncomment this line. Thats just how out of synch our fork is with main and how undermaintained our fork is.

Reply:
I saw that too...the problem is that downstream repos expect this behavior and broke when I "fixed" this json lib  🤦🏻   so upgrading is a bigger task.  let's scope down to just 429 errors and start there